domain/modelmanager/service/service.go

```diff
@@ -45,6 +52,6 @@ func (s *Service) Delete(ctx context.Context, uuid UUID) error {
 		return errors.Annotatef(err, "validating model uuid %q", uuid)
 	}
 
-	err := s.st.Delete(ctx, uuid)
+	err := s.dbManager.DeleteDB(uuid.String())
```

Comment:
For symmetry with `Create`, I would rather:
- `Delete` accepts `DeleteDB` as a closure.
- The removal from the table is done here.
- The accessor worker deletes the tracker worker upon invocation of the closure as its *only* responsibility.